cmd/secretless-broker/main.go

```diff
@@ -2,12 +2,48 @@ package main
 
 import (
 	"flag"
+	"os"
+	"os/signal"
+	"syscall"
 
-	"github.com/cyberark/secretless-broker/internal/plugin"
 	"github.com/cyberark/secretless-broker/pkg/secretless"
+	v2 "github.com/cyberark/secretless-broker/pkg/secretless/config/v2"
+	"github.com/pkg/profile"
 )
 
 func main() {
+	params := CmdLineParams()
+
+	// TODO: pass in logger, config, eventNotifier, what else?
+
+	var test []*v2.Service
+
+	// checksum logic
+
+	// allAvailablePlugins
+
+	// validate requested plugins
+
+	cfg := v2.Config{
+		Debug: false,
+		Services: test,
+	}
+
+	secretless := NewSecretless()
```

Comment:
This will need some params from CLI so we might need to keep an options struct that we pass into it